src/test/java/org/symphonyoss/symphony/messageml/MessageMLContextTest.java

```diff
@@ -116,6 +116,18 @@ public void testParseMessageML() throws Exception {
     assertEquals("Legacy entities", MAPPER.writeValueAsString(expectedEntities), MAPPER.writeValueAsString(entities));
   }
 
+  @Test
```

Comment:
was this test failing locally when you ran it without your fix? (maybe not because you are already using utf-8)
to make sure it works we could try to enforce the encoding before the test (and reset it after)

Reply:
in order to change the default encoding we should change the file.encoding property which needs to specified at JVM starts up (changing it with System.setProperty("file.encoding","US-ASCII") won't do anything at this point).

Reply:
Ok a bit tricky to handle then, let's not do it